eng/configurations/targetgroups.props

```diff
@@ -122,18 +122,23 @@
       <CompatibleWith>netstandard</CompatibleWith>
     </TargetGroups>
     <TargetGroups Include="netcoreapp3.0">
-      <NETCoreAppMaximumVersion>3.0</NETCoreAppMaximumVersion>
       <TargetFramework>netcoreapp3.0</TargetFramework>
       <Imports>netcoreapp2.2</Imports>
       <CompatibleWith>netstandard2.1</CompatibleWith>
     </TargetGroups>
-    <!-- netcoreapp is an alias for netcoreapp3.0 -->
-    <TargetGroups Include="netcoreapp">
-      <NETCoreAppMaximumVersion>3.0</NETCoreAppMaximumVersion>
-      <TargetFramework>netcoreapp3.0</TargetFramework>
+    <TargetGroups Include="netcoreapp3.1">
+      <NETCoreAppMaximumVersion>3.1</NETCoreAppMaximumVersion>
```

Comment:
I believe we can remove `NETCoreAppMaximumVersion` here if we set it in Directory.Build.targets